platform/ios/src/MGLMapView.h

```diff
@@ -1622,7 +1622,7 @@ MGL_EXPORT
  @note In versions prior to `4.0.0` selecting an offscreen annotation did not
  change the camera.
  */
-- (void)selectAnnotation:(id <MGLAnnotation>)annotation animated:(BOOL)animated;
+- (void)selectAnnotation:(id <MGLAnnotation>)annotation animated:(BOOL)animated __attribute__((deprecated("Use `-selectAnnotation:animated:completionHandler:` instead.", "selectAnnotation:animated:completionHandler:")));
```

Comment:
Is it safe to provide a replacement string? Or will Xcode’s fix-it suggestion get confused because there’s an additional parameter at the end?

Reply:
I suppose this isn’t strictly useful, yeah — Xcode inserts the replacement before the existing parameters (and doesn’t just add `completion:` at the end). I’ll remove these and file a ~radar~ feedback with Apple.